plugins/AuthUI/build.gradle.kts

```
@@ -0,0 +1,57 @@
+plugins {
+    alias(libs.plugins.kotlin.multiplatform)
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.compose)
+}
+
+description = "Extends gotrue-kt with ui composables"
+
+repositories {
+    mavenCentral()
+    maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+}
+
+@OptIn(org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi::class)
+kotlin {
+    targetHierarchy.default()
+    androidTarget {
+        publishLibraryVariants("release", "debug")
+    }
+
```

```suggestion
    targetHierarchy.default()
    jvmToolchain(8)
    androidTarget {
        publishLibraryVariants("release", "debug")
    }

```
You may need to add `jvmToolchain` here, otherwise the build keeps failing